MAUI/Themes/Keys.md

```diff
@@ -294,4 +294,1497 @@ This page lists the keys for each control and the element to which it is mapped
+        <tr>
+        <td>
+            SfLinearGaugeBarPointerFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the pointer fill of the linear gauge.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	  <tr>
+        <td>
+            SfLinearGaugeLabelTextColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the label text.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	       <tr>
+        <td>
+            SfLinearGaugeMajorTickStroke  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the major tick stroke.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+		       <tr>
+        <td>
+            SfLinearGaugeMinorTickStroke  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the minor tick stroke.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+		       <tr>
+        <td>
+            SfLinearGaugeAxisLineFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the axis line fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+		       <tr>
+        <td>
+            SfLinearGaugeRangeFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the range fill.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+		       <tr>
+        <td>
+            SfLinearGaugeShapePointerFillColor  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the shape pointer of the gauge.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            SfLinearGaugeShapePointerStroke  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the pointer stroke
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	 <tr>
+        <td>
+            SfLinearGaugeNormalBackground  
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of the gauge background.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+	</tr>
+</table>
+
+
+## SfStepProgressBar
+
+<table>
+    <tr>
+        <th>Theme Dictionary<br/>
+            <br/></th>        
+        <th>
+          Keys
+            <br/>
+            <br/>
+        </th>
+        <th>
+            Description
+            <br/>
+            <br/>
+        </th>
+    </tr>
+    <tr>
+        <td rowspan="55">
+           SfStepProgressBarThemeStyles
+            <br/>
+            <br/>
+        </td>
+		<td>
+           SfStepProgressBarTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfStepProgressBar without merging common theme resource and control style resource dictionaries.
```

Comment:
By merging this key in application resources, you can customize the appearance of SfStepProgressBar without merging common theme resources and control style resource dictionaries.